.github/workflows/ci.yml

```diff
@@ -12,6 +12,13 @@ env:
   NODE_OPTIONS: --max-old-space-size=6144
 
 jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v5
```

Comment:
suggestion: give this step a descriptive name
```suggestion
      - name: Lint commit messages
        uses: wagoid/commitlint-github-action@v5
```

Reply:
@LayZeeDK Done, thanks!